api/spec/controllers/mno_enterprise/jpi/v1/app_instances_sync_controller_spec.rb

```diff
@@ -38,7 +38,7 @@ module MnoEnterprise
       HashWithIndifferentAccess.new(name: 'a_name', status: 'FAILED', date: nil)
     ] }
 
-    let!(:organization_with_connectors) { build(:organization, connectors: connectors) }
+    let!(:organization_with_connectors) { build(:organization, connectors: connectors, has_running_cube: false) }
```

Comment:
Could worth adding a short test case for when `org.has_running_cube = true`: you can make sure `has_running_cube` is returned by the controller, so that if we break it with a later change, the spec will turn red..